docs/README.md

```diff
@@ -1,42 +1,30 @@
-docs-superstaq: Home of documentation for Superstaq
+docs: Home of documentation for Superstaq
 ===================================================
-This repository contains materials that support the [Superstaq documentation site](https://docs-superstaq.readthedocs.io/).
+This repository contains materials that support the [Superstaq documentation site](https://superstaq.readthedocs.io/en/latest/).
 
-## How to build docs-superstaq locally
+## How to build the docs locally
 ### Setup your environment
 
-Clone the repository, set up your virtual environment, install requirements, and make sure submodules are up-to-date.
+Clone the repository, set up your virtual environment, and install requirements.
 
-    git clone git@github.com:Infleqtion/docs-superstaq.git
-    python3 -m venv docs-superstaq-env
-    source docs-superstaq-env/bin/activate
-    cd docs-superstaq
+    git clone git@github.com:Infleqtion/client-superstaq.git
+    python3 -m venv venv_superstaq
+    source venv_superstaq/bin/activate
+    cd client-superstaq/docs
     pip install -r requirements.txt
-    git submodule update --init --recursive
+    
 
 ### Build the docs
-1. `cd` into the `docs` folder
-0. `make clean`
-0. `make html`
+1. `cd` into the `checks` folder
+0.  Run the script `build_docs.py`
```

Comment:
usually we suggest running e.g. `checks/build_docs.py` from the parent dir, instead of cd-ing into `checks/`